week-1/Homework/mandatory/3-function-output.js

```diff
@@ -1,31 +1,43 @@
 // Add comments to explain what this function does. You're meant to use Google!
 function getNumber() {
-    return Math.random() * 10;
+  return Math.random() * 10;
+  // The Math.random() function returns a floating - point, pseudo - random number in the range 0 to less than 1(inclusive of 0, but not 1) with approximately uniform distribution over that range — which you can then scale to your desired range.
```

Comment:
These are good definitions of what `Math.random` and `String.concat` do :) Could you also add a comment about what _these functions_ (`getNumber` and `s`) do, because they're slightly different?

For `getNumber`, one thing to think about is what the lowest and highest values it can return are, and include that in your comment :)